web/tools/webpack.config.babel.js

```diff
@@ -35,6 +35,43 @@ const readVersionName = () => {
   return versionFile.versionName
 }
 
+const generateManifest = (content: string, buildConfigName: string): string => {
+  const manifest = JSON.parse(content.toString())
+
+  const androidBuildConfig = loadBuildConfig(buildConfigName, ANDROID)
+  const commonBuildConfig = loadBuildConfig(buildConfigName, COMMON)
+
+  manifest.version = readVersionName()
+  manifest.homepage_url = commonBuildConfig.aboutUrls.default
+  manifest.theme_color = commonBuildConfig.lightTheme.colors.themeColor
+  manifest.related_applications = [{
+    platform: 'play',
+    id: androidBuildConfig.applicationId,
+    url: `https://play.google.com/store/apps/details?id=${androidBuildConfig.applicationId}`
+  }]
+
+  switch (buildConfigName) {
+    case 'integreat':
+    case 'integreat-test-cms':
+      manifest.name = 'Integreat'
```

Comment:
I think we don't have to hardcode this, should be included in the build config already as name. Regarding the description: Getting it from the locales is probably to complex, right?

Reply:
Manifest is generated, when building the app. So we don't know the locales of the user at this point.

Reply:
We could just german as default though as we do now as well I think? But probably the complexity with overrides is too much.

Reply:
We can keep it in mind, and change it if somebody ever encounters this string and wonders or wants them to be in the correct language. I do not even know where (if ever) they are displayed?